src/legacy/core_plugins/kibana/public/dashboard/grid/dashboard_grid.tsx

```diff
@@ -94,8 +104,7 @@ function ResponsiveGrid({
       draggableHandle={isViewMode ? '.doesnt-exist' : '.dshPanel__dragger'}
       layout={layout}
       onLayoutChange={onLayoutChange}
-      measureBeforeMount={false}
-      onResize={ensureWindowScrollsToBottom}
+      onResize={({}, {}, {}, {}, event) => ensureWindowScrollsToBottom(event)}
```

Comment:
removed measureBeforeMount because it appeared to be non-functional. might be worth a closer look.